packages/components/psammead-navigation/src/index.stories.jsx

```diff
@@ -0,0 +1,196 @@
+import React, { Fragment } from 'react';
+import styled from 'styled-components';
+import { storiesOf } from '@storybook/react';
+import { select, number, withKnobs } from '@storybook/addon-knobs';
+import { inputProvider, dirDecorator } from '@bbc/psammead-storybook-helpers';
+import { latin } from '@bbc/gel-foundations/scripts';
+import * as svgs from '@bbc/psammead-assets/svgs';
+import VisuallyHiddenText from '@bbc/psammead-visually-hidden-text';
+import Brand from '@bbc/psammead-brand';
+import Navigation, { NavigationUl, NavigationLi } from './index';
+import igboNavData from '../testHelpers/igbo';
+import pidginNavData from '../testHelpers/pidgin';
+import yorubaNavData from '../testHelpers/yoruba';
+import notes from '../README.md';
+
+const inputs = () => {
+  // capitalization is only for presentation purpose on the knob
+  const options = Object.keys(svgs)
+    .filter(key => key !== 'BBC_BLOCKS')
+    .map(key => key.charAt(0).toUpperCase() + key.slice(1));
+
+  const svgChoice = select('Service SVG', options, 'News').toLowerCase();
+  const svgRatio = svgs[svgChoice].ratio;
+  const svgMaxHeight = 24;
+  const svgMinHeight = 16;
+  const minWidthInput = number('minimum svg width', svgRatio * svgMinHeight);
+  const maxWidthInput = number('maximum svg width', svgRatio * svgMaxHeight);
+  const svgHeightInput = number('desired height svg', svgMaxHeight);
+
+  return { svgChoice, svgHeightInput, minWidthInput, maxWidthInput };
+};
+
+const StyledMain = styled.main`
+  padding: 0px 1rem;
+`;
+
+const offScreenText = (
+  <Fragment>
+    <span
+      // eslint-disable-next-line jsx-a11y/aria-role
+      role="text"
+    >
+      <span lang="en-GB">BBC News</span>, Ìgbò - Akụkọ
+    </span>
+  </Fragment>
+);
+
+storiesOf('Components|Navigation', module)
+  .addDecorator(withKnobs)
+  .add(
```

Comment:
Could this file be 1/3 the size? If you loop over an object, the stories can be generated with a reused function

Reply:
agreed with Drew. Looks like the last two stories are pretty much identical
![image](https://user-images.githubusercontent.com/13763899/59846213-26913880-9357-11e9-8400-cc6e6ee8b52e.png)